Sources/MapboxNavigation/NavigationMapView+Annotations.swift

```diff
@@ -30,60 +30,69 @@ extension NavigationMapView {
-            // define the "content" area of the image which is the portion that the maps sdk will use
-            // to place the text label within
-            let imageContent = ImageContent(left: 34, top: 32, right: 56, bottom: 50)
-            
-            let regularAnnotationImage = image.tint(routeDurationAnnotationColor)
-            try style.addImage(regularAnnotationImage,
-                               id: "RouteInfoAnnotationRightHanded",
-                               stretchX: stretchX,
-                               stretchY: stretchY,
-                               content: imageContent)
-            
-            let selectedAnnotationImage = image.tint(routeDurationAnnotationSelectedColor)
-            try style.addImage(selectedAnnotationImage,
-                               id: "RouteInfoAnnotationRightHanded-Selected",
-                               stretchX: stretchX,
-                               stretchY: stretchY,
-                               content: imageContent)
+        let routeDurationAnnotationImageIdentifier: String
+        switch tailPosition {
+        case .leading:
+            routeDurationAnnotationImageIdentifier = ImageIdentifier.routeAnnotationLeftHanded
+        case .trailing:
+            routeDurationAnnotationImageIdentifier = ImageIdentifier.routeAnnotationRightHanded
         }
         
-        // Left-hand pin
-        if let image = Bundle.mapboxNavigation.image(named: "RouteInfoAnnotationLeftHanded") {
-            // define the "stretchable" areas in the image that will be fitted to the text label
-            // These numbers are the pixel offsets into the PDF image asset
-            let stretchX = [ImageStretches(first: Float(47), second: Float(48))]
-            let stretchY = [ImageStretches(first: Float(28), second: Float(32))]
-            // define the "content" area of the image which is the portion that the maps sdk will use
-            // to place the text label within
-            let imageContent = ImageContent(left: 47, top: 28, right: 52, bottom: 40)
-            
-            let regularAnnotationImage = image.tint(routeDurationAnnotationColor)
-            try style.addImage(regularAnnotationImage,
-                               id: "RouteInfoAnnotationLeftHanded",
-                               stretchX: stretchX,
-                               stretchY: stretchY,
-                               content: imageContent)
-            
-            let selectedAnnotationImage = image.tint(routeDurationAnnotationSelectedColor)
-            try style.addImage(selectedAnnotationImage,
-                               id: "RouteInfoAnnotationLeftHanded-Selected",
-                               stretchX: stretchX,
-                               stretchY: stretchY,
-                               content: imageContent)
-        }
+        let capInsets = UIEdgeInsets(top: 15.0, left: 15.0, bottom: 35.0, right: 15.0)
```

Comment:
This approach allows to stop using magic numbers and make placement similar across different sceens (iPad, iPhone with/without notch etc). For example:

![Simulator Screen Shot - iPad Air 2 - 2022-12-05 at 15 43 58](https://user-images.githubusercontent.com/1496498/205777012-e0f1adc7-d8bc-4cbd-a2bb-01f92868b1e8.png)